app/models/elastic_search_recommended_link.rb

```diff
@@ -23,29 +24,19 @@ def body
 
   def details
     {
-      title: @recommended_link.title,
-      link: @recommended_link.link,
-      description: @recommended_link.description,
+      title: recommended_link.title,
+      link: recommended_link.link,
```

Comment:
Very minor comment: since `link` is being delegated to `recommended_link`, you could replace this with `link`.